<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>橘星－烟花软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource/images.qrc">
    <normaloff>:/Images/title.png</normaloff>:/Images/title.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/Images/main.png);</string>
  </property>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>连接测试</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFire">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>手动燃放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProject">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>工程编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>470</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resource/images.qrc"/>
 </resources>
 <connections/>
</ui>
